uat/custom-components/client-python-paho/README.md

```diff
@@ -0,0 +1,81 @@
+# Test MQTT5/311 client bases on Paho for Python
+
+The controlled `test MQTT v5.0/v3.1.1 client` is based on Paho Python is used to test Greengrass v2 MQTT v5.0 compatibility.
+
+## Setup
+
+Install requirements
+
+```cmd
+pip install -r src/requirements.txt
+```
+
+Install linting and checking requirements
+
+```cmd
+pip install -r src/requirements_lint.txt
+```
+
+## Run Client
+
+### Generate gRPC files
+
+Windows
+
+```cmd
+scripts\\generate_grpc.bat
+```
+
+Linux
+
+```sh
+./scripts/generate_grpc.sh
+```
+
+### Arguments
+Currently that client support three arguments and only first is mandatory.
+
+#### Agent Id
+Arbitrary string which will identify that instance of client in Control.
+
+#### IP of control
+The IP address where gRPC server of Control is listening.
+Value 127.0.0.1 will be used by default.
+
+#### Port of control
+The TCP port where gRPC server of Control is listening.
+Value 47619 will be used by default.
+
+### Run
+
+```cmd
+python src/main.py agent1 47619 127.0.0.1

```

Comment:
Replace agent1 to paho-python-agent. 
That will works in both cases but can avoid mistakes